src/actions/LoginActions.ts

```diff
@@ -11,43 +10,41 @@ enum LoginActionKeys {
 }
 
 export interface LoginPayload {
-  logged?: boolean;
-  sessionTimeOut?: number;
-  token: Token;
-  username: string;
-}
-
-export interface LoginFailurePayload {
-  error: any;
+  status: LoginStatus;
+  session?: Session;
+  error?: any;
 }
 
 // synchronous action creators
 export const LoginActions = {
   loginRequest: createAction(LoginActionKeys.LOGIN_REQUEST),
-  loginExtend: createAction(
-    LoginActionKeys.LOGIN_EXTEND,
-    resolve => (token: Token, username: string, currentTimeOut: number) =>
-      resolve({
-        token: token,
-        username: username,
-        sessionTimeOut: currentTimeOut + config().session.extendedSessionTimeOut
-      } as LoginPayload)
+  loginExtend: createAction(LoginActionKeys.LOGIN_EXTEND, resolve => (session: Session) =>
+    resolve({
+      status: LoginStatus.loggedIn,
+      session: session,
+      error: undefined
+    } as LoginPayload)
   ),
```

Comment:
The configuration about the session is for Kiali session in the UI not for the client, we shouldn't remove this. The ticket of openshift is 24 hours by default.

Reply:
What do you mean by that? When we extend the token duration, how do we update it?

Reply:
With another attribute, session was created to control the time of the user in the UI not in the backend, openshift in this case. This was a requirement by UX , maybe we should ask to @kiali/ux-reviewers and @abonas about this.

Is this working only in openshift or you test this in Kubernetes ?

Reply:
I did test it on kubernetes but only for `anonymous` and `login` strategies, as kubernetes does not provide oauth. Is that what you mean?

Reply:
@aljesusg Can you please update your comments with what your concern is here? What is changing in the UX aspect here?

Reply:
The session attribute in the UI was controlled by config().session.sessionTimeOut, it was a timeout for the UI, independent of the backend token, this was a request to kick the user from the UI of they have not activity, so my concern is about this. On the other hand is if we remove this session to have all the control in the openshift token, How is going to work for Kiali in Kubernetes without oauth?

Reply:
We don't support Oauth in anything other than OpenShift. For k8s, we use the username and password login.

Reply:
Ok but with this we are removing the session of the UI, this is my concern, if we are going to control de UI session from the backend or not, there is a pop-up timeout in the UI that it's working with this.